eframe/Cargo.toml

```diff
@@ -49,19 +51,23 @@ screen_reader = [
   "tts",
 ]
 
+# Use WGPU as the backend instead of glow
+wgpu = ["egui-wgpu"]
+
 
 [dependencies]
 egui = { version = "0.18.0", path = "../egui", default-features = false, features = [
   "bytemuck",
   "tracing",
 ] }
-egui_glow = { version = "0.18.0", path = "../egui_glow", default-features = false }
-glow = "0.11"
 tracing = "0.1"
 
 # optional:
+egui_glow = { version = "0.18.0", path = "../egui_glow", optional = true, default-features = false }
```

Comment:
While randomly going through this `Cargo.toml`, there's a `puffin = ["dep:puffin", "egui_glow/puffin"]` that enables this optional `egui_glow` now. I think the feature is only used to emit profiling scopes in that crate, should it perhaps be updated to `egui_glow?/puffin`?

Reply:
good catch; will fix

Reply:
fixed in d76d055d49e99c4c5f4c85ee6ab19b92e14951f6

Reply:
Thanks! I still had a little entry for this on my TODO list, can be crossed off now :)